config/configurator.go

```diff
@@ -0,0 +1,44 @@
+package config
+
+import "github.com/spf13/viper"
+
+type configurationLoader interface {
```

Comment:
Minor naming thing: I know "Loader" handles environment setup and it seems to be fine, but "Reader" actually "loads" configs from file to struct, so "Reader" sounds more like a "Loader". But in this case I'm not sure what we should call "Loader" - both "Initializer" and "Configer" sound weird to me. But again, naming is minor.

Reply:
This is for future discussion.